### DISCUSSION TITLE
Would a SAS Card Work?

### DISCUSSION THREAD

**Discussion by BzowK** [mebs/CM4-NAS#11] [Q&A]
Good Morning - 

LOVE the idea and design.  Currently have an 8 disk pool in my PC which I've put off putting into a dedicated NAS due to cost, but this may be solution.  Only question is if possible to connect x8 disks.  I currently use a [RocketRAID 2720 SAS PCIe card](https://www.amazon.com/High-Point-RocketRAID-2720A-8-6GB/dp/B07RJV2BHH/ref=sr_1_1?dchild=1&keywords=2720+sas&qid=1617888254&sr=8-1) to connect all x8 disks (using two x4 SATA breakout cables) and the [specs](https://highpoint-tech.com/USA_new/series_rr272x_specification.htm) say it's PCIe 2.0 x8 interface plus has [Linux drivers](https://highpoint-tech.com/USA_new/series_rr272x_configuration.htm), but don't know if it would be compatible with this or another configuration.

Any thoughts?  Thanks and great work!

**Comment by mebs** [comment]
Yes it's possible! You will find this very interesting: https://www.youtube.com/watch?v=Zpfq8ZC2hyI&t

Keep in mind that the CM4 only offers a single lane of PCIe 2.0, which is limiting for such a demanding use case.

**Reply by BzowK** [reply]
Great - Thanks!
